app/views/provider/admin/onboarding/wizard/request/show.html.slim

```diff
@@ -26,12 +26,14 @@ ol.explain.fa-ul
     i.icon.fa-li.fa.fa-puzzle-piece.icon--success
     .text
       strong> = link_to @request.api_name, edit_provider_admin_onboarding_wizard_api_path
-      ' returned <a class="toggle" href="#response">a response</a> to the customer through the gateway.
+      ' returned <a id="request-toggle" href="#response">a response</a> to the customer through the gateway.
 
 = link_to "Cool, what's next?", provider_admin_onboarding_wizard_outro_path, class: "button"
 
-coffee:
-  $(".toggle").live "click", ->
-    target = $(this).attr("href")
-    $(target).toggleClass("is-hidden");
-
+javascript:
+  document.addEventListener('DOMContentLoaded', () => {
+    const toggle = document.getElementById('request-toggle')
+    toggle.addEventListener('click', () => {
+      document.getElementById('response').classList.toggle('is-hidden')
```

Comment:
we don't rely on running a css selector on `a href` but hardcode the `id`. That's alright I guess.

Reply:
Yeah, I thought it was kind of weird to take it from `href` attribute, if we know that what we want to toggle is the `#response` block.

Reply:
Even if the href changed, the id would not 😄 I think it's nonsense and as @mayorova  said, it's not supposed to be dynamic.